pkg/cli/server/root.go

```diff
@@ -752,6 +756,54 @@ func LoadConfiguration(config *config.Config, configPath string) error {
 	return nil
 }
 
+func updateLDAPConfig(conf *config.Config) error {
+	if conf.HTTP.Auth == nil || conf.HTTP.Auth.LDAP == nil {
+		return nil
+	}
+
+	if conf.HTTP.Auth.LDAP.Unauthenticated {
+		conf.HTTP.Auth.LDAP.SetBindDN("anonym-user")
+
+		return nil
+	}
+
+	if conf.HTTP.Auth.LDAP.CredentialsFile == "" {
```

Comment:
If the credentials file is empty we should assume we need to do an anonymous bind to search for users instead?